scripts/run-benchmarks.py

```diff
@@ -0,0 +1,185 @@
+import subprocess
+import json
+import os
+import re
+
+import click
+import cpuinfo
+
+BASE_REPO_URL = 'https://github.com/lumol-org/lumol.git'
+
+INDIVIDUAL_BENCH_TEMPLATE = """
+<details><summary>%s %s</summary>
+  <p>\n\n
+\n\n
+```bash\n
+%s\n\n
+```\n\n
+
+</p></details>
+"""
+
+COMPARISON_TEMPLATE = """
+<details><summary>%s %s</summary>
```

Comment:
Could you use 'new' string formatting, using `"{}".format()`? It was added in [Python 2.6](https://docs.python.org/2/library/string.html#custom-string-formatting), so it is always available.